hydra_python_core/doc_maker.py

```diff
@@ -8,7 +8,7 @@
 from typing import Any, Dict, Match, Optional, Tuple, Union
 
 
-def error_mapping(body: str=None) -> str:
+def error_mapping(body: str = None) -> str:
```

Comment:
Is this PEP8 compliant?
https://stackoverflow.com/questions/8853063/pep-8-why-no-spaces-around-in-keyword-argument-or-a-default-parameter-value

Reply:
Ok, I didn't see that, but it passes the test. Also I used AutoPEP8 to get this output.